src/EditorFeatures/CSharp/Wrapping/BinaryExpression/CSharpBinaryExpressionWrapper.cs

```diff
@@ -13,7 +13,7 @@ public CSharpBinaryExpressionWrapper()
         {
         }
 
-        protected override SyntaxTriviaList GetNewLineBeforeOperatorTrivia(SyntaxTriviaList newLine)
+        public override SyntaxTriviaList GetNewLineBeforeOperatorTrivia(SyntaxTriviaList newLine)
```

Comment:
required because i moved the abstract method to a locatoin that isn't accessible through protected access.

Reply:
I still maintain that a code fix to expand accessibility would be useful in practice...